tajo-docs/src/main/sphinx/functions/hivefunc.rst

```diff
@@ -0,0 +1,81 @@
+##############
+Hive Functions
+##############
+
+Tajo provides a feature to use Hive functions directly without re-compilation or additional code.
+
+*************
+Configuration
+*************
+
+Only thing to do is registering path to a directory for jar files containing your hive functions.
+You can do this by set ``tajo.function.hive.code-dir`` in ``tajo-site.xml`` like the following.
```

Comment:
```hive.jar-dir``` would be more proper for this config. I know this is out of scope of this issue. But, we need to improve it before releasing.